Shared/Features/Freaks/View/FreaksView.swift

```diff
@@ -21,6 +21,7 @@ struct FreaksView: View {
             }
             .navigationBarTitle("Freaks", displayMode: .inline)
         }
+        .accentColor(Color("SecondaryColor"))
```

Comment:
Let's set the `accentColor` at the app level. In this way, we won't need to do the same setup on every view.